Sources/MuxPlayerSwift/ReverseProxyServer/ReverseProxyServer.swift

```diff
@@ -16,22 +16,22 @@ class ReverseProxyServer {
 
     }
 
-    class ManifestReversifier {
+    class PlaylistLocalURLMapper {
```

Comment:
Looking at this again, `PlaylistLocalURLMapper` makes even more sense. Especially if read without context. Great idea 👍